src/main/java/mainpack/MainRun.java

```diff
@@ -24,82 +24,69 @@ public void incomeMenu(){
     public static void main(String[] args) {
         MainRun menu = new MainRun();
         Database database = new Database();
+        InOutHndler handler = new InOutHndler();
         Scanner scanner = new Scanner(System.in);
         int choice, index;
-        menu.mainMenu();
-        choice = scanner.nextInt();
-        while (choice < 6){
-            switch(choice){
-                case 1:
-                    Client tmp = new Client();
-                    tmp.addClient();
-                    database.getList().add(tmp);
-                    System.out.println("Client successfully added");
-                    break;
-                case 2:
-                    if(database.getList().isEmpty()){
-                        System.out.println("There are no clients in the database");
-                    } else {
+        do{
+            menu.mainMenu();
+            choice = scanner.nextInt();
+            if(choice == 1){
+                Client tmp = new Client();
+                handler.addClient(tmp);
+                database.getList().add(tmp);
+                System.out.println("Client successfully added");
+            }
+            else if(choice == 2 || choice == 3 || choice == 4 || choice == 5){
```

Comment:
Can this expression be simplified with more and less operators?

Reply:
fixed